tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -91,9 +77,19 @@ void GenerateConditionVariableBasedOnGitChange(string variableName, string[] fil
                     else
                     {
                         var changedFiles = GetGitChangedFiles(baseBranch);
-
                         // Choose changedFiles that meet any of the filters => Choose changedFiles that DON'T meet any of the exclusion filters
-                        isChanged = changedFiles.Any(s => filters.Any(filter => s.StartsWith(filter, StringComparison.OrdinalIgnoreCase)) && !exclusionFilters.Any(filter => s.Contains(filter, StringComparison.OrdinalIgnoreCase)));
+
+                        var teamName = _isChangedTeam[variableName];
+                        foreach (var changedFile in changedFiles)
+                        {
+                            var match = codeOwners.Match("/" + changedFile);
+                            if (match?.Owners.Contains(teamName) == true)
```

Comment:
is there any reason in particular we are explicitly presenting the equality? 🙂
```suggestion
                            if (match?.Owners.Contains(teamName))
```

Reply:
correct, sorry I missed that part

Reply:
No problem! :)